clarity_ext/dilution.py

```diff
@@ -72,7 +82,7 @@ def _build_plate_sorting_map(containers):
         # Fetch an unique list of container names from input
         # Make a dictionary with container names and plate position sort numbers
         unique_containers = sorted(list(
-            {container.id for container in containers}))
```

Comment:
I should be `container.id`, not `container.resource.id`. But this is an outdated diff, if you look at the discussion thread in Github, it says "GitEdvard commented on an outdated diff 7 minutes ago". I believe that this is happening because you're looking at individual commits. It might be better to look at "Files changed" instead. (I personally only review each commit in turn if the pull request is too large.)